user-journey-tests/helpers/testHelpers/finalAnswers.js

```diff
@@ -0,0 +1,49 @@
+import destinationAnswersPage from '../../page-objects/destination/destinationAnswersPage'
+import finalAnswersPage from '../../page-objects/finalAnswersPage'
+import landingPage from '../../page-objects/landingPage'
+import checkAnswersPage from '../../page-objects/origin/checkAnswersPage'
+import licenceAnswersPage from '../../page-objects/receiving-the-licence/licenceAnswersPage'
+import taskListPage from '../../page-objects/taskListPage'
+import completeDestinationTask from './destination'
+import { completeOriginTaskAnswersCustom } from './movementLicence'
+import { completeLicenceTaskAnswersCustom } from './receivingLicence'
+
+export const completeApplication = async (originObject, licenceObject) => {
```

Comment:
We probably should allow all sections to be passed in as arguments with some default values if not provided